src/main/java/com/easypost/service/CarrierAccountService.java

```diff
@@ -30,13 +31,18 @@ public class CarrierAccountService {
      * @return created CarrierAccount object.
      * @throws EasyPostException when the request fails.
      */
-    public CarrierAccount create(final Map<String, Object> params)
-            throws EasyPostException {
+    public CarrierAccount create(final Map<String, Object> params) throws EasyPostException {
+        String type = (String) params.get("type");
+        if (type == null) {
+            throw new EasyPostException("Carrier account type is required");
```

Comment:
You should use `MissingParameterError` here to be more explicit on the error type. There is a constant you can use `MISSING_REQUIRED_PARAMETER`

Reply:
Resolved